.github/workflows/pages.yml

```diff
@@ -43,15 +43,8 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
-      - name: Select build directory
-        run: |
-          CABAL_BUILDDIR="dist-newstyle"
-
-          echo "CABAL_BUILDDIR=$CABAL_BUILDDIR"
-          echo "CABAL_BUILDDIR=$CABAL_BUILDDIR" >> $GITHUB_ENV
-
       - name: Install Haskell
-        uses: input-output-hk/setup-haskell@v1
+        uses: input-output-hk/setup-haskell@v2
```

Comment:
There seems to be no `@v2` tag? https://github.com/input-output-hk/setup-haskell/tags